data/Parvej2001.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "Parvej Shaikh",
+  "displayStatsPublic" : true,
+  "type":"personal",
+  "bio": "Building Axstudios.tech and Skilled in Full Stack development.Exploring Cloud Native and Blockchain technology with a bachelor of engineering focused in Information Technology",
+  "avatar": "https://github.com/parvej2001.png",
+  "links": [
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/p_parvez",
+      "icon": "FaTwitter"
+    },
+    {
+      "name": "Lets collobrate here",
```

Comment:
I think there is a typo on word `collaborate`

Reply:
Thank you 😊 for letting me know sir
I will correct it 😄